src/main.cpp

```diff
@@ -3045,6 +3051,7 @@ bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockIndex* pin
 
             std::vector<CScriptCheck> vChecks;
             unsigned int flags = SCRIPT_VERIFY_P2SH | SCRIPT_VERIFY_DERSIG;
+
```

Comment:
Could (should) be doing a check against supermajority within this function to append to the `flags` variable the `SCRIPT_VERIFY_CHECKLOCKTIMEVERIFY` status.